app-modules/case/database/migrations/2023_03_06_000008_create_case_items_table.php

```diff
@@ -6,21 +6,21 @@
 
 class CreateCaseItemsTable extends Migration
 {
-    public function up()
+    public function up(): void
     {
         Schema::create('case_items', function (Blueprint $table) {
-            $table->bigIncrements('id');
+            $table->uuid('id')->primary();
             $table->integer('casenumber');
             $table->string('respondent_type')->nullable();
             $table->string('respondent_id')->nullable();
             $table->longText('close_details')->nullable();
             $table->longText('res_details')->nullable();
-            $table->unsignedBigInteger('institution_id')->nullable();
-            $table->unsignedBigInteger('status_id')->nullable();
-            $table->unsignedBigInteger('type_id')->nullable();
-            $table->unsignedBigInteger('priority_id')->nullable();
-            $table->unsignedBigInteger('assigned_to_id')->nullable();
-            $table->unsignedBigInteger('created_by_id')->nullable();
+            $table->uuid('institution_id')->nullable();
```

Comment:
We can probably do this in another PR, but makes sense to use `foreignUuid()` here and remove the extra migration to define the foreign key constraints. Will just need to re-order migrations.

Reply:
Yeah I can rearrange these at a later PR